README.MD

```diff
@@ -4,31 +4,37 @@ This is a set of tools developed to aid in the tracking of maptick, td, and pop
 
 It scrapes all available performance logs, processes them into csv files (One for each map, one for each server, and a few global ones), and then uses `gnuplot` to plot them in a few different ways I've judged as useful.
 
-## Scraping
+## Setup
 
-We don't scrape all performance logs, since some of them from before 2020/11/11 don't scrape well and don't match the current system.
+### Install Dependencies
 
-## Parsing
+Download and install the following:
+- [Python 3.6](https://www.python.org/downloads/release/python-360/)
+- [gnuplot 5+](http://www.gnuplot.info/)
 
-A brief note on parsing, we compile each round into one data-point, attached to some other info about it.
+Make sure both are in your path, then restart your computer. You should then run `Setup.bat`, this will setup the default header config in [root/config](root/config) and download any python modules needed through pip.
```

Comment:
why do you need to restart for this?

Reply:
Personally I had to restart to get the path changes to propagate, and this is recommended for path changes in general.

Reply:
I was under the impression that you just needed to open a new terminal to refresh the path

Reply:
I was running these not through a terminal but instead by double clicking the batch files themselves, doing it through terminal might have worked. Want me to replace the restart mention with a recommendation to use a terminal?

Reply:
It's odd that it what you describe wouldn't just work then. I thought the only restriction was needing to refresh a terminal's copy of environment vars after modifying them. are you sure a restart is needed in the situation you describe?

Reply:
No I'm not but even the windows dialogue itself recommends restarting when you edit the path variable, which is why I repeated it here after it had fixed my own issue.

Reply:
huh. ok I guess this is fine then